src/challeng.c

```diff
@@ -20,12 +20,11 @@
 #define NAME_SIZE 30
 
 /* Local function defs.  Created independantly in interp.h Uggh. */
-void do_challenge args((CHAR_DATA * ch, char *argument));
-void do_accept args((CHAR_DATA * ch, char *argument));
-void do_refuse args((CHAR_DATA * ch, char *argument));
```

Comment:
we lost refuse here... but I see it's defined in interp.h (ugh indeed whoever wrote 'Uggh'). Long term I will be breaking merc.h into smaller pieces to make it tractable too, so challeng.h can be the canonical source. (well, challengE.h as we're not limited by DOS filename lengths any more...)

Reply:
Yeah I removed them intentionally as they weren't needed.